app/components/ConnectorItem.jsx

```diff
@@ -5,13 +5,24 @@ import { translate } from '../plugins/preact-polyglot'
 
 const ConnectorItem = ({ title, subtitle, connected, slug, iconName, backgroundCSS = 'white', enableDefaultIcon = false, router }) => (
   <Link class='item-wrapper' to={`${router.location.pathname}/${slug}`}>
+    {backgroundCSS &&
```

Comment:
Indeed, but backgroundCSS recieves the content of initKonnectors.json, where I emptied the color.css value.